### ISSUE TITLE
Errors in running test `pytest cage-challenge-1/CybORG/CybORG/Tests/test_sim`

### ISSUE THREAD

**Issue by aduyinuo** [cage-challenge/cage-challenge-1#6]
<img width="1680" alt="Screen Shot 2021-11-09 at 1 38 43 AM" src="https://user-images.githubusercontent.com/40970294/140875025-420329e7-8292-47d2-8c15-bdd2058b3e95.png">

I cloned the latest version of CybORG following the instructions in install.md. 

To reproduce the error:
```
conda create --name cage python=3.8
conda activate cage
pip install numpy-stl
pip install -e cage-challenge-1/CybORG
pytest cage-challenge-1/CybORG/CybORG/Tests/test_sim
```

What am I missing here? Thanks!

**Comment by mavroudisv** [comment]
I submitted a pull request for the above. However, there are still some more errors.

I tried to fix the follow-up errors with a quick check (couldn't tell if the anomaly_dict is None due to another error in the calling function though):

``    def _process_anomalies(self,anomaly_dict):
        info = deepcopy(self.blue_info)
        if anomaly_dict:
          for hostid, host_anomalies in anomaly_dict.items():
``

But there are still assertion issues. For quick debugging:

``pytest cage-challenge-1/CybORG/CybORG/Tests/test_sim -k test_BlueTable -vv``
